tests/benchmarks/test_join.py

```diff
@@ -10,46 +11,48 @@ def memory_multiplier(request):
 
 
 @run_up_to_nthreads("small_cluster", 40, reason="Does not finish")
-def test_join_big(small_client, memory_multiplier, configure_shuffling):
-    memory = cluster_memory(small_client)  # 76.66 GiB
+def test_join_big(small_client, memory_multiplier, shuffle_method):
+    with dask.config.set({"dataframe.shuffle.method": shuffle_method}):
```

Comment:
For some reason using the `configure_shuffling` fixture here breaks things on CI.